packages/esm-patient-chart-app/src/view-components/side-menu.component.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { useEffect } from "react";
+import { SideNav } from "carbon-components-react/es/components/UIShell";
+import { SideNavProps } from "carbon-components-react";
+import { ExtensionSlot, useLayoutType } from "@openmrs/esm-framework";
+import styles from "./side-menu.component.scss";
+import { isTablet } from "../utils";
+
+interface SideMenuPanelProps extends SideNavProps {}
+
+const LAYOUT_CSS_CLASS = "omrs-sidenav-expanded";
+
+const SideMenuPanel: React.FC<SideMenuPanelProps> = () => {
+  const layout = useLayoutType();
+
+  useEffect(() => {
```

Comment:
That should not be in here.

In general this seems like its equivalent to the omrs layout classes we have already. Otherwise, I'd argue that this class should be taken care of from primary nav. Its not the responsibility of the patient chart to determine the layout of the whole application.

Reply:
yea, i was kind of assuming this would be a hack. i was wondering if it would just make more sense for the sidebar on patient charts to not be position absolute since this is the only component that would be using a sidebar on desktop

Reply:
sorry, will work on this story - just gonna see if making the navbar local to the patients and wihtout a posiiton absolute so i can avoid this useEffect whilst making the navbar truly `local`

Reply:
https://github.com/openmrs/openmrs-esm-patient-chart/pull/217/commits/46ee7b05289b5f617ee159ccf918b1689a528a46 just using local styling to achieve the same result. so i got rid of the useeffect in this case.

Reply:
@jonathandick @FlorianRappl